rudder-core/src/test/resources/configuration-repository/expected-share/root-default-install/common/1.0/cf-served.cf

```diff
@@ -59,13 +59,13 @@ bundle server access_rules
 
     any::
           "/var/rudder/share/root/"
-            maproot    => { host2ip("server.rudder.local"), string_downcase(escape("server.rudder.local"))  },
-            admit      => { host2ip("server.rudder.local"), string_downcase(escape("server.rudder.local"))  },
+            maproot    => { string_downcase(escape("server.rudder.local"))  },
```

Comment:
That change should not be linked with the modification at hand, but I'm not sure why I need it. I'm wondering if we didn't change something elsewhere.

Reply:
strange, if not something in your PR then tests must failed but they passs ...

Reply:
I know, and I don't know what to think about it. @ncharles, do you have an idea?

Reply:
Found why, it's because we changed booleans in string template to string, which change conditionals behavior in stringtemplate